README.md

```diff
@@ -42,3 +42,60 @@ See the [local-env/README.md](./local-env/README.md) for more details on the loc
 Please be aware that the productive deployment of the control-api may run on a different Kubernetes distribution than [kind].
 
 [kind]: https://kind.sigs.k8s.io/
+
+
+### Running the control-api locally
+
+You can run the control-api locally against the currently configured Kubernetes cluster with
+
+```bash
+make run
+```
+
+To access the locally running API server you need to register it with the [kind]-based local environment.
+You can do this by applying the following.
+
+```yaml
+---
+apiVersion: apiregistration.k8s.io/v1
+kind: APIService
+metadata:
+  name: v1.organization.appuio.io
```

Comment:
I wouldn't put too much effort into automating this as only someone working on the API server itself needs this and only once, but I'll try to make it a bit more copy-pastable

Reply:
What do you mean "only once"? How would I be able to debug an issue with an IDE?

Reply:
I mean only once in the sense of you apply it once to your local test cluster and as long as you do not delete the cluster you don't have to do it again.

Reply:
Hm, I often delete local clusters at the end of the day to save resources on the workstation and to generally clean up. I'd love some automation here especially since those clusters are meant to be brought up and thrown away often.

Reply:
Hmm oke. I just stop and restart them :sweat_smile:

Reply:
Alright I added some sed magic. Should be completely copy-pastable.

Reply:
Thanks :)